app/views/edit/deployment-config.html

```diff
@@ -356,6 +356,11 @@ <h3 class="with-divider">Environment Variables</h3>
                         </div>
                       </div>
 
+                      <pause-rollouts-checkbox
+                          deployment="updatedDeploymentConfig"
+                          always-visible="true">
```

Comment:
was this because you can change the triggers within this page?  why not show/hide this depending on whether the dc trigger is enabled?